app/middlewares/auth.js

```diff
@@ -0,0 +1,25 @@
+const errors = require('../errors'),
+  userService = require('../services/users'),
+  sessionManager = require('../services/sessionManager'),
+  moment = require('moment');
+
+exports.isAuthenticated = (req, res, next) => {
+  const usersToken = req.headers[sessionManager.HEADER_NAME];
+  if (usersToken) {
+    const user = sessionManager.decode(usersToken);
```

Comment:
I find it a little bit confusing naming this `user` I would have gone with something more like tokenContent or tokenPayload, or something more descriptive, since the result of decoding the jwt isn't the user.